mantidproject/mantid
# Tiled plotting for raw muon data

**Description of work.**
This is to make it easier for users to quickly inspect their raw data for unusual behaviour (e.g. broken or faulty detectors). 



**To test:**


Test fitting and plotting in muon analysis in general

To test the raw plots:
Load some data into muon analysis
In the plot window change the plot to "Raw Plot"
You will now see a plot of counts
You can turn it into a tiled plot

Check the above works for single and multi period.

Partial fix  for #29887 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
